webpack.config.js

```diff
@@ -2,6 +2,7 @@
 const path = require('path');
 
 module.exports = {
+	mode: 'production',
```

Comment:
Using `'development'` outputs nicer names in stack traces than `'production'` IIRC. The other change you'd need if you take this suggestion is to the [`build-renderer` script in `package.json`](https://github.com/getflywheel/local-addon-image-optimizer/blob/master/package.json#L27)

```json
{
  "scripts": {
    "build-renderer": "NODE_ENV=\"production\" yarn webpack --config webpack.config.js"
  }
}
```

```suggestion
	mode: process.env.NODE_ENV ===  'production' ? 'production' : 'development',
```